src/routes/movies/index.tsx

```diff
@@ -129,8 +137,7 @@ const Movies: FC = () => {
                 className='mainAnimatedPage libraryPage backdropPage collectionEditorPage pageWithAbsoluteTabs withTabs'
                 backDropType='movie'
             >
-                {getTabComponent(selectedIndex)}
-
+                {getTabComponent(searchTabIndex === -1 ? tabManagerIndex : searchTabIndex)}
```

Comment:
There isn't currently a state variable that contains this value since the `tabManagerIndex` is updated by the `tabManager`... should I add another state variable for this?

Reply:
@thornbill 
https://user-images.githubusercontent.com/32230989/224406574-9c642908-bdda-4a81-9f1d-fd29a1c7e71b.mp4

Reply:
Yeah... I think I'll have to fix this by changing the mainTabManager (which I was trying to avoid) :sweat_smile: